packages/node_modules/@node-red/runtime/lib/module/template/package.json.mustache

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "{{&module}}",
+    "version": "{{&version}}",
+    "description": "{{&desc}}",
+    "node-red": {
+        "nodes": {
+            "{{&type}}": "subflow.js"
+        }
+    },
+    "keywords": {{&keywords}},
+    "dependencies": {
+        "crypto-js": "^4.0.0"
```

Comment:
Why does this now depend on `crypto-js`? *edit* - ah, see my comment from `subflow.js`